soccer/src/referee/ExternalReferee.hpp

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <rj_protos/LogFrame.pb.h>
+#include <rj_protos/referee.pb.h>
+
+#include <boost/asio.hpp>
+#include <boost/config.hpp>
+#include <cstdint>
+#include <mutex>
+#include <rclcpp/rclcpp.hpp>
+#include <rj_common/RefereeEnums.hpp>
+#include <rj_common/Utils.hpp>
+#include <rj_msgs/msg/raw_protobuf.hpp>
+#include <thread>
+#include <vector>
+
+#include "GameState.hpp"
+#include "Node.hpp"
+#include "RefereeBase.hpp"
+#include "SystemState.hpp"
+#include "TeamInfo.hpp"
+#include "config_client/config_client.h"
+
+namespace referee {
+
+using RawProtobufMsg = rj_msgs::msg::RawProtobuf;
+
+/**
+ * @brief The ref module listens to a port for referee packets over the network.
+ *
+ * @details ExternalReferee packets are sent out from the
+ * [ssl-refbox](https://github.com/Hawk777/ssl-refbox) program.
```

Comment:
This is the old refbox though... maybe link the new one at https://github.com/RoboCup-SSL/ssl-game-controller?

Reply:
I mean I think we still use the refbox mostly, although we should probably move over